javascript/javascript3/week2/index.js

```diff
@@ -0,0 +1,96 @@
+//Movies exercise
+const movieUrl =
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
+async function movieFunction() {
```

Comment:
Try and seperate the api call from the data manipulation

Reply:
Api call separated from the data manipulation